### DISCUSSION TITLE
Modoboa LDAP Auth Problem with WS2008 / Samba4

### DISCUSSION THREAD

**Discussion by DaTaru** [modoboa/modoboa#2216] [Help] [answered]
Hello,

I have a Problem with the LDAP Auth on my Modoboa Server.
When i use a Windows Server 2016 as ADDC i can login with an AD user.
But when i use a Windows Server 2008R2 or a Samba4 ADDC i cant. 
With my Samba ADDC i can login into Windows and Linux with winbind. I can even use the backend on Nextcloud to login.

Where can i find a log to find the problem with the connection? 
Has anybody run into the same issue?

**Comment by tonioo** [comment, marked as answer]
To get more information, enable debug traces (see https://django-auth-ldap.readthedocs.io/en/latest/logging.html).

**Reply by DaTaru** [reply]
I've enabled the debug for Django and now i see this in the logs:

`Caught LDAPError while authenticating user1@XXX.XXX: INVALID_CREDENTIALS({'msgtype': 97, 'msgid': 1, 'result': 49, 'desc': 'Invalid credentials', 'ctrls': [], 'info': '80090308: LdapErr: DSID-0C0903A9, comment: AcceptSecurityContext error, data 52e, v1db1'})
`

**Reply by DaTaru** [reply]
**edit**

I got it to work by changing the Bind user from the Default Administrator to an another user.
After that the bind was successful and i just had to change the search filter to "(mail=%(user)s)" to find my users.

Thanks for the reply, tonioo
